lib/tasks/temporary/resave_all_articles_and_comments.rake

```diff
@@ -0,0 +1,9 @@
+namespace :temporary do
+  desc "Resave all articles and comments"
```

Comment:
User-generated content such as articles and comments will need to be resaved because the generated image URLs are "hardcoded" in the DB. Resaving will trigger `Images::Optimizer` to use Imgproxy instead of Cloudinary.

Reply:
Talked in Slack about making this a DataUpdateScript to ensure it runs on all environments.